### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by divyanshthinks**
Use modular JavaScript to create a web app for adding and removing books from a list.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+#author {
+  margin-bottom: 15px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (franclobo) [inline index.css]: - [ ] Kindly give styles to your pages your end result should look similar to this: 👇🏽 

| **Original design** | **Student design** |
|---|---|
| image | image |
| image | image |
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @divyanshthinks, 

# Required Changes ♻️
Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please 🙏🏽 add the MIT LICENSE file to your README file and make sure that the link works as expected for a professional README file. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (Mihndim2020) [inline README.md:36]: - [OPTIONAL] It will be nice to remove all sections of your readme file that is unrelated to your project as this will make your readme file clean and professional. I mean sections on which you actually have information at the moment, but update the sections if information is available. Also remember to update your project title.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @divyanshthinks, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
